NuGet.config

```diff
@@ -3,7 +3,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <!--TODO: Merged PIAs - evaluate the need of this once packages are in public feeds-->
+    <!-- TODO: Keep in-sync with build\import\Packages.targets -->
```

Comment:
nit:

```suggestion
    <!-- NOTE: Keep in-sync with build\import\Packages.targets -->
```

To me, TODO implies some pending task that can be completed, whereas this message is a caution for ongoing changes.

Reply:
Fixed.